### PR TITLE
Opossum reporter fun test fixes

### PR DESCRIPTION
**Pull request by fviernau**
See individual commits.

Part of #5950.

### CODE REVIEW

File: plugins/reporters/opossum/src/test/kotlin/OpossumReporterTest.kt [hunk 1]
```diff
@@ -150,15 +150,18 @@ class OpossumReporterTest : WordSpec({
             val signals = opossumInput.getSignalsForFile(
                 "/pom.xml/compile/first-package-group/first-package@0.0.1/LICENSE"
             )
+
```
Comment (sschuberth) [inline plugins/reporters/opossum/src/test/kotlin/OpossumReporterTest.kt:153]: Nit: Unrelated formatting changes here and below.

**Review by sschuberth — COMMENTED** [review]
LGTM. @maxhbr are you also fine with these changes?

**Review by mnonnenmacher — APPROVED** [review]
